examples/regional-network-firewall-policy/main.tf

```diff
@@ -0,0 +1,216 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  prefix   = "regional"
+  location = "us-central1"
+}
+
+data "google_project" "project" {
+  project_id = var.project_id
+}
+
+resource "random_string" "random_suffix" {
+  length  = 6
+  special = false
+  lower   = true
+  upper   = false
+}
+
+resource "google_compute_network" "network" {
+  project = var.project_id
+
+  name = "${local.prefix}-network"
+}
+
+resource "google_tags_tag_key" "tag_key" {
+
+  description = "For keyname resources."
+  parent      = "projects/${data.google_project.project.number}"
```

Comment:
Do we need number? Usually project ids are also supported.

Reply:
TPG recreates the resource, on every execution, when I use project ID. I created a [bug](https://github.com/hashicorp/terraform-provider-google/issues/14792) for the provider. They are removing project ID from the documentation.